src/freenas/usr/local/libexec/freenas-debug/network/network.sh

```diff
@@ -126,59 +85,71 @@ network_func()
 	done
 	section_footer
 
-	section_header "Default Route"
-	if is_linux; then
-		ip route show default | awk '/default/ {print $3}'
-	else
-		route -n show default|grep gateway|awk '{ print $2 }'
+	# grab the interfaces marked critical for failover
+	# if this is an HA system
+	is_ha=$(midclt call failover.licensed)
+	if [ "$is_ha" = "True" ]; then
+		section_header "Interfaces marked critical for failover"
+		ints=$(${FREENAS_SQLITE_CMD} ${FREENAS_CONFIG} -line "
+		SELECT
```

Comment:
Should we use `midclt` to retrieve this information please like we are doing above perhaps ? I think that would mean consistent results as well like via midclt and what we expect in debug causing less confusion between field names etc as well ( not saying this is confusing but just saying that would be consistent with the API as well ) ?

Reply:
Using midclt makes it impossible to read since 1 interface is 111 lines long after passing it to jq.

Reply:
We can use `select` to only retrieve values we are interested in ?

Reply:
I'd rather not. Parsing becomes a headache and the debug scripts don't really have a standard that is followed. 😄